website/src/components/ImageMasker.vue

```diff
@@ -125,12 +125,43 @@ export default {
 			);
 		},
 		drawMask(mask) {
```

Comment:
I think the drawmask functionality should remain as it was, perhaps we need a computed value `defaultMaskPosition` that uses `imageData`.

It's important we don't confuse the functionality of working out the bounding box and rotation of the ideal mask with the functionality of overlaying one PNG on another (what `drawMask` currently does).